scripts/suite.py

```diff
@@ -109,6 +109,9 @@
                               When tests finish or time out, send an email
                               here. May also be specified in ~/.teuthology.yaml
                               as 'results_email'
+  --rocket <rocket>           Comma separated list of rocketchat channels where
+                              to send a message when tests finished or time out.
+                              To be used with --sleep-before-teardown option.
```

Comment:
Hm, I don't like `--rocket`. I would prefer `--rocketchat` or `--rocket-chat` because the software is called "Rocket.Chat" (not "Rocket"). The word "rocket" can mean several things (including a vegetable), but people not familiar with "Rocket.Chat" will simply be confused by an option called `--rocket`.

Therefore, please consider renaming the option to `--rocketchat` or `--rocket-chat`?

Reply:
I go with `--rocketchat`, thanks for the suggestion.